xml/hardening.xml

```diff
@@ -2164,7 +2164,8 @@ FIXME:
      Also, other packages like FTP and Telnet daemons should not be installed
      as well unless there is a justified business reason for it (again,
      <command>ssh</command>, <command>scp</command> or <command>sftp</command>
-     should be used as replacements).
```

Comment:
Nice to have: Move the parenthesis into a new sentence, remove the word "again".